demo/core_34_$.www.code.js

```diff
@@ -179,8 +179,10 @@ $.www.code.editor.www = function(request, response) {
       if (request.parameters.src) {
         var ok = true;
         try {
-          // TODO(fraser): Make this secure -- somehow.
```

Comment:
Did you mean to remove this TODO?

Reply:
That TODO basically applies to the entirety of the database, so it's a bit misleading on that specific line.